setup.py

```diff
@@ -16,14 +16,17 @@
     "PyYAML>=4.2b1",
 ]
 
+# docker is needed to use moto 3.0.0 and above
+# see https://github.com/spulec/moto/issues/3567#issuecomment-947047013
 dev_requires = [
     "black==22.3.0",
+    "docker>=5.0.0",
```

Comment:
This requirement should be installed by moto

Reply:
true, got it deleted in [12062ed](https://github.com/Skyscanner/cfripper/pull/219/commits/12062ed53e819209ee3c996a7134aadf0738a5ec)